contrib/mesos/docs/scheduler.md

```diff
@@ -0,0 +1,115 @@
+# Kubernetes-Mesos Scheduler
+
+## Labels and Mesos Agent Attributes
+
+The scheduler of Kubernetes on Mesos takes [labels][1] into account: it matches
+specified labels in pod specs with defined labels of nodes.
+
+In addition to user defined labels, [attributes of Mesos agents][2] are converted
+into node labels by the scheduler, following the pattern
+
+```yaml
+k8s.mesosphere.io/attribute-<name>: value
+```
+
+As an example, an attribute `generation: 2015` will result in the node label
+
+```yaml
+k8s.mesosphere.io/attribute-generation: 2015
+```
+
+and be used to schedule pods onto node which are of generation 2015.
+
+**Note:** Node labels which originate from Mesos agent attributes must not be
```

Comment:
Suggestion:
Node labels prefixed by `k8s.mesosphere.io` are managed by Kubernetes-Mesos and should not be modified manually by the user or admin. For example, the Kubernetes-Mesos executor manages `k8s.mesosphere.io/attribute` labels and will auto-detect and update modified attributes when mesos-slave is restarted.